Jenkinsfile

```diff
@@ -7,8 +7,8 @@ def jobParams = [
 
 def appPipline = new org.ios.AppPipeline(
     steps: this, 
-    appTests: false,
+    appTests: true,
     appPushNoti: true,
     jobParams: jobParams,
-    label: 'macos-ios-1-2')
+    label: 'macOS-latest')
```

Comment:
Why did you change it? We does not have runner called macOS-latest. Build will be stuck because of it

Reply:
Bao said me change it to macos-latest. and it works previously. Anyway I've changed back